src/Makefile

```diff
@@ -19,7 +22,12 @@ NETCDF ?= 1
 # Compile flags
 $(info DEBUG setting: '$(DEBUG)')
 ifeq ($(DEBUG), 1)
-  FCFLAGS := -g -Wall -Wextra -Wconversion -Og -pedantic -fcheck=bounds -fmax-errors=5
+  ifeq ($(FC),gfortran)
+    FCFLAGS := -g -Wall -Wextra -Wconversion -Og -pedantic -fcheck=bounds -fmax-errors=5 
+  else ifeq ($(FC),ifort)
+    #FCFLAGS := -g -warn all -implicitnone -O1 -stand f90 -error-limit 5  #Equivalent to gfortran Debug flags, builds with warnings on Hera
```

Comment:
Can we address those warnings as part of this PR?

Reply:
Agreed.

Reply:
@drnimbusrain  Surprisingly , it didn't gave any warnings with FCFLAGS := -g -warn all -implicitnone -O1 -stand f90 -error-limit 5  , this time
I used this code snippet though:
# Compiler
FC ?= gfortran
ifeq ($(FC), f77)  # override possible Make default
  FC := ifort
endif

Reply:
These are the warnings (used FC=ifort this time):

DEBUG=1 NETCDF=1 FC=ifort make -C src
FC setting: 'ifort'
DEBUG setting: '1'
NETCDF setting: '1'
LIBS: ' -L/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdff -L/apps/netcdf/4.6.1/intel/16.1.150/lib -Wl,-rpath -Wl,/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdf -lnetcdf'
INC:  ' -I/apps/netcdf/4.6.1/intel/16.1.150/include'
make: Entering directory `/scratch1/BMC/rcm2/qrasool/canopy-app/src'
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_const_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_coord_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_canopts_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_canmet_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_canvars_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_utils_mod.F90
canopy_utils_mod.F90(20): warning #8144: ASSOCIATE construct is not available in Fortran 90.
        associate(n => size(x))
--------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_files_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_readnml.F90
canopy_readnml.F90(56): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        CALL EXIT(istat)
-------------^
canopy_readnml.F90(212): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        CALL EXIT(istat)
-------------^
canopy_readnml.F90(219): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        CALL EXIT(istat)
-------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_alloc.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_init.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_txt_io_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_ncf_io_mod.F90
canopy_ncf_io_mod.F90(2051): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%lat=reshape(variables_2d%lat,[size(variables_2d%lat)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2059): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%lon=reshape(variables_2d%lon,[size(variables_2d%lon)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2069): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%clu=reshape(variables_2d%clu,[size(variables_2d%clu)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2078): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%csz=reshape(variables_2d%csz,[size(variables_2d%csz)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2087): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%ffrac=reshape(variables_2d%ffrac,[size(variables_2d%ffrac)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2096): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%fh=reshape(variables_2d%fh,[size(variables_2d%fh)])
-------------------------------------------------^
canopy_ncf_io_mod.F90(2105): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%frp=reshape(variables_2d%frp,[size(variables_2d%frp)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2114): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%href=reshape(variables_2d%href,[size(variables_2d%href)])
-----------------------------------------------------^
canopy_ncf_io_mod.F90(2123): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%lai=reshape(variables_2d%lai,[size(variables_2d%lai)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2132): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%mol=reshape(variables_2d%mol,[size(variables_2d%mol)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2141): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%fricv=reshape(variables_2d%fricv,[size(variables_2d%fricv)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2150): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%ugrd10m=reshape(variables_2d%ugrd10m,[size(variables_2d%ugrd10m)])
-----------------------------------------------------------^
canopy_ncf_io_mod.F90(2159): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%vgrd10m=reshape(variables_2d%vgrd10m,[size(variables_2d%vgrd10m)])
-----------------------------------------------------------^
canopy_ncf_io_mod.F90(2168): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%sfcr=reshape(variables_2d%sfcr,[size(variables_2d%sfcr)])
-----------------------------------------------------^
canopy_ncf_io_mod.F90(2177): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%vtype=reshape(variables_2d%vtype,[size(variables_2d%vtype)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2186): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%sotyp=reshape(variables_2d%sotyp,[size(variables_2d%sotyp)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2195): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%pressfc=reshape(variables_2d%pressfc,[size(variables_2d%pressfc)])
-----------------------------------------------------------^
canopy_ncf_io_mod.F90(2204): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%dswrf=reshape(variables_2d%dswrf,[size(variables_2d%dswrf)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2213): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%shtfl=reshape(variables_2d%shtfl,[size(variables_2d%shtfl)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2222): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%tmpsfc=reshape(variables_2d%tmpsfc,[size(variables_2d%tmpsfc)])
---------------------------------------------------------^
canopy_ncf_io_mod.F90(2231): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%tmp2m=reshape(variables_2d%tmp2m,[size(variables_2d%tmp2m)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2240): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%spfh2m=reshape(variables_2d%spfh2m,[size(variables_2d%spfh2m)])
---------------------------------------------------------^
canopy_ncf_io_mod.F90(2249): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%hpbl=reshape(variables_2d%hpbl,[size(variables_2d%hpbl)])
-----------------------------------------------------^
canopy_ncf_io_mod.F90(2258): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%prate_ave=reshape(variables_2d%prate_ave,[size(variables_2d%prate_ave)])
---------------------------------------------------------------^
canopy_ncf_io_mod.F90(943): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(951): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(959): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(982): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(990): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(998): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(1526): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1538): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1546): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1554): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1562): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1570): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1588): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1605): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1622): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1639): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1665): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1671): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1688): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1694): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1711): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1718): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1725): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1742): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1749): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1756): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1768): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit (2)
---------------------^
canopy_ncf_io_mod.F90(1850): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1869): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1891): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1913): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1926): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit (2)
---------------------^
canopy_ncf_io_mod.F90(2008): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_ncf_io_mod.F90(2019): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit(2)
-----------------^
canopy_ncf_io_mod.F90(2026): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit(2)
-----------------^
canopy_ncf_io_mod.F90(2036): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit(2)
-----------------^
canopy_ncf_io_mod.F90(2048): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2056): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2066): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2075): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2084): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2093): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2102): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2111): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2120): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2129): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2138): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2147): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2156): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2165): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2174): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2183): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2192): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2201): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2210): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2219): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2228): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2237): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2246): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2255): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2266): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2273): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2282): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2289): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2296): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2303): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2310): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2317): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2324): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2331): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2338): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2345): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2352): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2359): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2366): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2373): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2380): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2387): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2394): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2401): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2408): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2415): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2422): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2429): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2433): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_check_input.F90
canopy_check_input.F90(31): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        call exit(2)
-------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_read_txt.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_dxcalc_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_profile_mod.F90
canopy_profile_mod.F90(67): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(90): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(113): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(136): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(145): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_profile_mod.F90(305): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                call exit(2)
---------------------^
canopy_profile_mod.F90(309): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_wind_mod.F90
canopy_wind_mod.F90(79): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(1)
-----------------^
canopy_wind_mod.F90(96): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(1)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_waf_mod.F90
canopy_waf_mod.F90(68): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                call exit(2)
---------------------^
canopy_waf_mod.F90(78): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_waf_mod.F90(93): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_waf_mod.F90(98): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_phot_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_eddy_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_bioparm_mod.F90
canopy_bioparm_mod.F90(797): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_bioemi_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_calcs.F90
canopy_calcs.F90(45): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_calcs.F90(119): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                                    call exit(2)
-----------------------------------------^
canopy_calcs.F90(239): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_calcs.F90(260): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_calcs.F90(331): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                                call exit(2)
-------------------------------------^
canopy_calcs.F90(451): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                call exit(2)
---------------------^
canopy_calcs.F90(458): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        call exit(2)
-------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_write_txt.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_dealloc.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_app.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF canopy_const_mod.o canopy_coord_mod.o canopy_canopts_mod.o canopy_canmet_mod.o canopy_canvars_mod.o canopy_utils_mod.o canopy_files_mod.o canopy_readnml.o canopy_alloc.o canopy_init.o canopy_txt_io_mod.o canopy_ncf_io_mod.o canopy_check_input.o canopy_read_txt.o canopy_dxcalc_mod.o canopy_profile_mod.o canopy_wind_mod.o canopy_waf_mod.o canopy_phot_mod.o canopy_eddy_mod.o canopy_bioparm_mod.o canopy_bioemi_mod.o canopy_calcs.o canopy_write_txt.o canopy_dealloc.o canopy_app.o -o ../canopy  -L/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdff -L/apps/netcdf/4.6.1/intel/16.1.150/lib -Wl,-rpath -Wl,/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdf -lnetcdf  -I/apps/netcdf/4.6.1/intel/16.1.150/include
make: Leaving directory `/scratch1/BMC/rcm2/qrasool/canopy-app/src'

Reply:
`DEBUG=1 NETCDF=1 FC=ifort make -C src
FC setting: 'ifort'
DEBUG setting: '1'
NETCDF setting: '1'
LIBS: ' -L/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdff -L/apps/netcdf/4.6.1/intel/16.1.150/lib -Wl,-rpath -Wl,/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdf -lnetcdf'
INC:  ' -I/apps/netcdf/4.6.1/intel/16.1.150/include'
make: Entering directory `/scratch1/BMC/rcm2/qrasool/canopy-app/src'
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_const_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_coord_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_canopts_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_canmet_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_canvars_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_utils_mod.F90
canopy_utils_mod.F90(20): warning #8144: ASSOCIATE construct is not available in Fortran 90.
        associate(n => size(x))
--------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_files_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_readnml.F90
canopy_readnml.F90(56): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        CALL EXIT(istat)
-------------^
canopy_readnml.F90(212): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        CALL EXIT(istat)
-------------^
canopy_readnml.F90(219): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        CALL EXIT(istat)
-------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_alloc.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_init.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_txt_io_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_ncf_io_mod.F90
canopy_ncf_io_mod.F90(2051): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%lat=reshape(variables_2d%lat,[size(variables_2d%lat)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2059): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%lon=reshape(variables_2d%lon,[size(variables_2d%lon)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2069): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%clu=reshape(variables_2d%clu,[size(variables_2d%clu)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2078): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%csz=reshape(variables_2d%csz,[size(variables_2d%csz)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2087): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%ffrac=reshape(variables_2d%ffrac,[size(variables_2d%ffrac)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2096): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%fh=reshape(variables_2d%fh,[size(variables_2d%fh)])
-------------------------------------------------^
canopy_ncf_io_mod.F90(2105): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%frp=reshape(variables_2d%frp,[size(variables_2d%frp)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2114): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%href=reshape(variables_2d%href,[size(variables_2d%href)])
-----------------------------------------------------^
canopy_ncf_io_mod.F90(2123): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%lai=reshape(variables_2d%lai,[size(variables_2d%lai)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2132): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%mol=reshape(variables_2d%mol,[size(variables_2d%mol)])
---------------------------------------------------^
canopy_ncf_io_mod.F90(2141): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%fricv=reshape(variables_2d%fricv,[size(variables_2d%fricv)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2150): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%ugrd10m=reshape(variables_2d%ugrd10m,[size(variables_2d%ugrd10m)])
-----------------------------------------------------------^
canopy_ncf_io_mod.F90(2159): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%vgrd10m=reshape(variables_2d%vgrd10m,[size(variables_2d%vgrd10m)])
-----------------------------------------------------------^
canopy_ncf_io_mod.F90(2168): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%sfcr=reshape(variables_2d%sfcr,[size(variables_2d%sfcr)])
-----------------------------------------------------^
canopy_ncf_io_mod.F90(2177): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%vtype=reshape(variables_2d%vtype,[size(variables_2d%vtype)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2186): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%sotyp=reshape(variables_2d%sotyp,[size(variables_2d%sotyp)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2195): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%pressfc=reshape(variables_2d%pressfc,[size(variables_2d%pressfc)])
-----------------------------------------------------------^
canopy_ncf_io_mod.F90(2204): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%dswrf=reshape(variables_2d%dswrf,[size(variables_2d%dswrf)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2213): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%shtfl=reshape(variables_2d%shtfl,[size(variables_2d%shtfl)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2222): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%tmpsfc=reshape(variables_2d%tmpsfc,[size(variables_2d%tmpsfc)])
---------------------------------------------------------^
canopy_ncf_io_mod.F90(2231): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%tmp2m=reshape(variables_2d%tmp2m,[size(variables_2d%tmp2m)])
-------------------------------------------------------^
canopy_ncf_io_mod.F90(2240): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%spfh2m=reshape(variables_2d%spfh2m,[size(variables_2d%spfh2m)])
---------------------------------------------------------^
canopy_ncf_io_mod.F90(2249): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%hpbl=reshape(variables_2d%hpbl,[size(variables_2d%hpbl)])
-----------------------------------------------------^
canopy_ncf_io_mod.F90(2258): warning #5197: Fortran 90 does not allow array constructors in square brackets
            variables%prate_ave=reshape(variables_2d%prate_ave,[size(variables_2d%prate_ave)])
---------------------------------------------------------------^
canopy_ncf_io_mod.F90(943): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(951): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(959): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(982): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(990): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(998): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit (2)
-----------------^
canopy_ncf_io_mod.F90(1526): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1538): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1546): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1554): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1562): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1570): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(1588): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1605): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1622): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1639): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit(2)
-------------------------^
canopy_ncf_io_mod.F90(1665): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1671): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1688): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1694): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1711): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1718): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1725): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1742): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1749): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1756): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1768): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit (2)
---------------------^
canopy_ncf_io_mod.F90(1850): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1869): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1891): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1913): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    CALL exit (2)
-------------------------^
canopy_ncf_io_mod.F90(1926): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit (2)
---------------------^
canopy_ncf_io_mod.F90(2008): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_ncf_io_mod.F90(2019): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit(2)
-----------------^
canopy_ncf_io_mod.F90(2026): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit(2)
-----------------^
canopy_ncf_io_mod.F90(2036): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            CALL exit(2)
-----------------^
canopy_ncf_io_mod.F90(2048): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2056): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2066): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2075): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2084): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2093): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2102): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2111): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2120): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2129): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2138): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2147): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2156): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2165): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2174): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2183): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2192): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2201): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2210): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2219): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2228): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2237): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2246): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2255): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2266): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2273): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2282): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2289): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2296): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2303): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2310): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2317): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2324): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2331): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2338): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2345): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2352): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2359): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2366): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2373): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2380): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2387): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2394): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2401): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2408): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2415): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2422): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2429): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                CALL exit(2)
---------------------^
canopy_ncf_io_mod.F90(2433): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_check_input.F90
canopy_check_input.F90(31): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        call exit(2)
-------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_read_txt.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_dxcalc_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_profile_mod.F90
canopy_profile_mod.F90(67): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(90): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(113): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(136): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_profile_mod.F90(145): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_profile_mod.F90(305): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                call exit(2)
---------------------^
canopy_profile_mod.F90(309): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_wind_mod.F90
canopy_wind_mod.F90(79): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(1)
-----------------^
canopy_wind_mod.F90(96): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(1)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_waf_mod.F90
canopy_waf_mod.F90(68): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                call exit(2)
---------------------^
canopy_waf_mod.F90(78): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_waf_mod.F90(93): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_waf_mod.F90(98): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_phot_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_eddy_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_bioparm_mod.F90
canopy_bioparm_mod.F90(797): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_bioemi_mod.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_calcs.F90
canopy_calcs.F90(45): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_calcs.F90(119): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                                    call exit(2)
-----------------------------------------^
canopy_calcs.F90(239): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                    call exit(2)
-------------------------^
canopy_calcs.F90(260): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
            call exit(2)
-----------------^
canopy_calcs.F90(331): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                                call exit(2)
-------------------------------------^
canopy_calcs.F90(451): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
                call exit(2)
---------------------^
canopy_calcs.F90(458): warning #7416: Fortran 90 does not allow this intrinsic procedure.   [EXIT]
        call exit(2)
-------------^
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_write_txt.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_dealloc.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF  -I/apps/netcdf/4.6.1/intel/16.1.150/include -c canopy_app.F90
ifort -g -warn all -implicitnone -O1 -stand f90 -error-limit 5   -DNETCDF canopy_const_mod.o canopy_coord_mod.o canopy_canopts_mod.o canopy_canmet_mod.o canopy_canvars_mod.o canopy_utils_mod.o canopy_files_mod.o canopy_readnml.o canopy_alloc.o canopy_init.o canopy_txt_io_mod.o canopy_ncf_io_mod.o canopy_check_input.o canopy_read_txt.o canopy_dxcalc_mod.o canopy_profile_mod.o canopy_wind_mod.o canopy_waf_mod.o canopy_phot_mod.o canopy_eddy_mod.o canopy_bioparm_mod.o canopy_bioemi_mod.o canopy_calcs.o canopy_write_txt.o canopy_dealloc.o canopy_app.o -o ../canopy  -L/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdff -L/apps/netcdf/4.6.1/intel/16.1.150/lib -Wl,-rpath -Wl,/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdf -lnetcdf  -I/apps/netcdf/4.6.1/intel/16.1.150/include
make: Leaving directory `/scratch1/BMC/rcm2/qrasool/canopy-app/src'`